react-dropzone/react-dropzone.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for react-dropzone
```

Comment:
Wouldn't it be better to make that PR directly to the original fork and downstream it later?